lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -44,10 +44,16 @@ define((require, exports, module) ->
         return to
 
     popState = (currentState, stack) ->
-      top = stack.pop()
+      if not stack?
+        return 'start'
 
-      if stack?
+      stack.pop()
```

Comment:
Did you really mean to pop element and forget it? This code will return 'start', right?

```
stack = ['foo']
popState('bar', stack)
```

What's the point?